src/main/java/seedu/address/storage/JsonAdaptedProject.java

```diff
@@ -82,6 +82,18 @@ public JsonAdaptedProject(Project source) {
      * @throws IllegalValueException if there were any data constraints violated in the adapted project.
      */
     public Project toModelType() throws IllegalValueException {
+        if (projectName == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    ProjectName.class.getSimpleName()));
+        }
+
+        ProjectName projectNameConverted;
```

Comment:
Just a minor suggestion. I think convertedProjectName would be clearer here. Since there are many variables starting with "project" and "projectName"